src/NuGet.Core/NuGet.Commands/Strings.resx

```diff
@@ -896,4 +896,10 @@ For more information, visit https://docs.nuget.org/docs/reference/command-line-r
     <value>Package content hash validation failed for {0}. Expected: {1} Actual: {2}</value>
     <comment>0 - package ID, 1 - expected Sha, 2 - actual sha</comment>
   </data>
+  <data name="Error_CentralPackageVersions_VersionsNotAllowed" xml:space="preserve">
+    <value>Projects that use central package version management should not define the version on the PackageReference items but on the PackageVersion items: {0}.</value>
```

Comment:
```suggestion
    <value>Projects that use central package version management should define the version on the PackageVersion. The following packages have defined the version on the PackageReference items: {0}.</value>
```

We also need a comment explaining that `{0}` is the list of packages. I had to look it up.